node-perf-dash.k8s.io/ingress.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
please point to /v1 api :)

Reply:
@rikatz We are still running the 1.16 version: 
```shell
kubectl --context=gke_kubernetes-public_us-central1_aaa version 
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.4", GitCommit:"e87da0bd6e03ec3fea7933c4b5263d151aafd07c", GitTreeState:"clean", BuildDate:"2021-02-18T16:12:00Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.15-gke.7800", GitCommit:"cef3156c566a1d1a4b23ee360a760f45bfbaaac1", GitTreeState:"clean", BuildDate:"2020-12-14T09:12:37Z", GoVersion:"go1.13.15b4", Compiler:"gc", Platform:"linux/amd64"}
```
The `networking.k8s.io/v1` is only available since 1.19. :-)

Reply:
damn...we should probably discuss about updating AAA control plane to at least v1.19 if this is available as stable. Sorry for the wrong replacement then :)